### PR TITLE
Feature#11 forgot password UI

### LINKED ISSUE #11: Forgot Password  UI screen
# REQUIREMENTS
Forgot Password  UI ready for Rubi API conexion. 

# Acceptance Criteria
List all required acceptance criteria
- [ ] Nice UI.
- [ ] We re-use code in the components Form Input and Form Button

### PR DESCRIPTION
**Pull request by JorgeGarciaRa**
# Description
We already done the Forgot Password UI ready, for Rubi API conexion.
We structured our code with Prettier
Navigation still working successfully
We re-use code in the components Form Input and Form Button

  ## Screenshots 

https://user-images.githubusercontent.com/62182855/114755918-c899c600-9d1f-11eb-8771-4c8f76d52e33.mp4

### CODE REVIEW

File: AlertCovidApp/src/screens/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    initialValues: {
+      email: '',
+    },
+    onSubmit: (values) => {
+      let errors = {};
+
+      if (!values.email) {
+        errors.email = 'Insert your Email';
+      } else if (
+        !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)
```
Comment (fernandoamz) [inline AlertCovidApp/src/screens/ForgotPassword.js:25]: Move this regex to constants file

File: AlertCovidApp/src/screens/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    handleSubmit,
+  } = useFormik({
+    initialValues: {
+      email: '',
+    },
+    onSubmit: (values) => {
+      let errors = {};
+
+      if (!values.email) {
+        errors.email = 'Insert your Email';
```
Comment (fernandoamz) [inline AlertCovidApp/src/screens/ForgotPassword.js]: all text must be into a constants file

File: AlertCovidApp/src/utils/Constants/ForgotPasswordConstants.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    HomeScreen: 'HomeScreen',
+    Profile: 'Profile',
+    LogIn: 'Log In',
+    HistoryTest: 'HistoryTest',
+    HistoryTest2: 'HistoryTest2',
+    forgotPassword: 'Forgot Password',
+    LogInS:'LogIn',
+    reset: 'Reset Password',
+  };
+
```
Comment (fernandoamz) [inline AlertCovidApp/src/utils/Constants/ForgotPasswordConstants.js:19]: only one blank space should be at the end of the file

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by hearagmu — DISMISSED** [review]
Good job!

**Review by fernandoamz — APPROVED** [review]

**Review by GracielaJazmin — APPROVED** [review]

**Review by vmagana3 — APPROVED** [review]
Good Work!

**Review by hearagmu — APPROVED** [review]
Ok

**Pull request merged**